easybuild/easyconfigs/a/Armadillo/Armadillo-11.4.3-foss-2022b.eb

```
@@ -12,7 +12,10 @@ source_urls = ['https://sourceforge.net/projects/arma/files']
 sources = [SOURCELOWER_TAR_XZ]
 checksums = ['87603263664988af41da2ca4f36205e36ea47a9281fa6cfd463115f3797a1da2']
 
-builddependencies = [('CMake', '3.24.3')]
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('HDF5', '1.14.0', '', ('gompi', '2022b')),
+]
 
 dependencies = [
     ('Boost', '1.81.0'),
```

HDF5 is actually more than just a build dependency here, since `libarmadillo.so` is dynamically linked to `libhdf5.so` if HDF5 is available 

```suggestion
]

dependencies = [
    ('Boost', '1.81.0'),
    ('HDF5', '1.14.0'),
```